src/core/server/http/ssl_config.ts

```diff
@@ -101,7 +101,7 @@ export class SslConfig {
    */
   public getSecureOptions() {
     if (this.supportedProtocols === undefined || this.supportedProtocols.length === 0) {
-      return 0;
+      return cryptoConstants.SSL_OP_NO_TLSv1;
```

Comment:
__question:__ sorry for the dumb question, just out of curiosity, should we also include `SSL_OP_NO_SSLv2` and/or  `SSL_OP_NO_SSLv3`? A bit confused by OpenSSL docs:

> Clients should avoid creating "holes" in the set of protocols they support, when disabling a protocol, make sure that you also disable either all previous or all subsequent protocol versions. In clients, when a protocol version is disabled without disabling all previous protocol versions, the effect is to also disable all subsequent protocol versions.

Reply:
Aha, interesting, thanks for the link!